src/main/java/net/openhft/chronicle/wire/AbstractCommonMarshallable.java

```diff
@@ -21,7 +21,10 @@
 
 /**
  * This uses bytes marshallable, non self describing messages by default.
+ *
+ * @deprecated This will become package-private in x.24 use {@link SelfDescribingMarshallable} or {@link BytesInBinaryMarshallable} instead
  */
+@Deprecated(/* to be made package-private in x.24 */)
 public abstract class AbstractCommonMarshallable implements Marshallable, BytesMarshallable {
```

Comment:
I like this very much but we will need to socialise more broadly within the team as we have generated code "in the wild" that uses this.

Normally we give 2 releases warning i.e. x.25